packages/x-components/tests/e2e/cucumber/related-tags/related-tags.spec.ts

```diff
@@ -1,27 +1,55 @@
 import { And, Given, Then, When } from 'cypress-cucumber-preprocessor/steps';
+import { createRelatedTagStub } from '../../../../src/__stubs__/related-tags-stubs.factory';
 
 let relatedTagsList: string[] = [];
 const secondRelatedTagsList: string[] = [];
 
+// Background
+Given('a related tags API with a known response', () => {
+  cy.intercept('https://api.empathy.co/getRelatedTags', req => {
+    req.reply({
+      relatedTags: [
+        createRelatedTagStub('lego', 'marvel', false, ''),
+        createRelatedTagStub('lego', 'bombero', false, 'marvel'),
+        createRelatedTagStub('lego', 'policia', false, 'bombero')
+      ]
+    });
+  }).as('interceptedRelatedTags');
+});
+
+Given('a related tags API with a selected one', () => {
+  cy.intercept('https://api.empathy.co/getRelatedTags', req => {
+    req.reply({
+      relatedTags: [
+        createRelatedTagStub('lego', 'bombero', false, ''),
+        createRelatedTagStub('lego', 'policia', false, 'bombero'),
+        createRelatedTagStub('lego', 'barbie', true, 'policia')
+      ]
+    });
+  }).as('interceptedRelatedTags');
```

Comment:
You're giving the same alias to both of the `intercept`'s so the second one is overriding the first, give the second one a different name.

```suggestion
  }).as('interceptedRelatedTagsWithSelection'); // Quick suggestion
```

- [ ] Give a new alias